src/core/components/layout/components/columns/styles.ts

```diff
@@ -1,22 +1,25 @@
 import { css } from '@emotion/react';
 import { space } from '@guardian/src-foundations';
-import { Breakpoint, from, until } from '@guardian/src-foundations/mq';
+import { Breakpoint, from, until, between } from '@guardian/src-foundations/mq';
+
+type ColumnBreakpoint = {
+	totalColumns: number;
+	rule: string;
+};
 
 export const columns = css`
 	box-sizing: border-box;
 	display: flex;
-	& > :last-of-type {
-		margin-right: 0;
-	}
+	margin-right: -${space[5]}px;
 	& > * {
 		margin-right: ${space[5]}px;
 	}
 `;
 
 const collapseBelowSpacing = css`
 	display: block;
+	margin-right: 0;
 	& > * {
-		margin-right: 0;
```

Comment:
I think we want to keep this zero margin for columns when collapsed

Reply:
Sorted!